docs/scos/dev/front-end-development/202212.0/oryx/building-components/component-implementation.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Oryx: Integrate Oryx Components in any web frameworks"
+description: Oryx Components are build as web components
+last_updated: Sept 12, 2023
+template: concept-topic-template
+---
+
+Oryx components are _framework agnostic_. This means that components can be used in other web frameworks.
+
+Oryx components are build as [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components). Web components are a suite of standard web technologies, widely embraced by browser vendors. The purpose of web components is to provide components in isolation, so that they can easily integrate with other web technologies.
+
+Web components can be built by different frameworks or even with plain html, css and javascript. Oryx components are implemented with [Lit](https://lit.dev). Lit is a lightweight open source framework from Google that can be used to build highly efficient web components. If you do not want to use Lit, you can use your own framework of choice to create web components, see [component customization](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-components/component-customization.html).
+
+<!--
+Web components have a number of specific characteristics that have an impact on the implementation.
```

Comment:
This commented block is not visible in the final document - do we need it?

Reply:
removed